srcpkgs/topgrade/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'topgrade'
 pkgname=topgrade
-version=3.3.0
+version=3.4.0
 revision=1
 build_style=cargo
 short_desc="Meta upgrade tool for pip, flatpak, your distro and everything else"
 maintainer="jcgruenhage <jan.christian@gruenhage.xyz>"
 license="GPL-3.0-only"
-homepage="https://github.com/r-darwish/topgrade"
-distfiles="https://github.com/r-darwish/topgrade/archive/v${version}.tar.gz"
-checksum=22f8065d3fe715726de400927f845323b66e93ca374fbb3b6276289f83052384
+homepage="https://github.com/r-darwish/${pkgname}"
```

Comment:
Please don't do this, it is very useful to have the url for the homepage as plain text in the template.

Reply:
Okay, sure. I saw a few packages doing this and thought it looks a bit less redundant.

Reply:
fixed in c028e57